tv_apps/smart-system/js/app_usage_metrics.js

```diff
@@ -430,17 +432,42 @@
     switch (e.type) {
 
     case APPOPENED:
-    case HOMESCREEN:
-      // The user has opened an app, switched apps, or switched to the
-      // homescreen. Record data about the app that was running and then
+      // The user has opened an app or switched apps.
+      // Record data about the app that was running and then
       // update the currently running app.
-      this.metrics.recordInvocation(this.getCurrentApp(),
-                                    now - this.getCurrentStartTime());
+      if (this._appIsUnderlay) {
```

Comment:
Please add some comments to describe when this condition will happen (as we discussed).